tests/Tags/Concerns/QueriesConditionsTest.php

```diff
@@ -106,17 +106,19 @@ public function it_filters_by_in_condition()
         $this->makeEntry('cat')->set('type', 'feline')->save();
         $this->makeEntry('lion')->set('type', 'feline')->save();
         $this->makeEntry('horse')->set('type', 'equine')->save();
+        $this->makeEntry('horse')->set('type', 'equine')->save();
```

Comment:
Looks like `$this->makeEntry('horse')->set('type', 'equine')->save();` is there twice. I'm guessing it only needs to be there once?

Reply:
Correct, probably duplicated the line by mistake!